tests/baselines/reference/mappedTypeWithAsClauseAndLateBoundProperty2.errors.txt

```diff
@@ -1,11 +0,0 @@
-tests/cases/compiler/mappedTypeWithAsClauseAndLateBoundProperty2.ts(1,14): error TS4118: The type of this node cannot be serialized because its property '[iterator]' cannot be serialized.
-tests/cases/compiler/mappedTypeWithAsClauseAndLateBoundProperty2.ts(1,14): error TS4118: The type of this node cannot be serialized because its property '[unscopables]' cannot be serialized.
-
-
-==== tests/cases/compiler/mappedTypeWithAsClauseAndLateBoundProperty2.ts (2 errors) ====
-    export const thing = (null as any as { [K in keyof number[] as Exclude<K, "length">]: (number[])[K] });
-                 ~~~~~
-!!! error TS4118: The type of this node cannot be serialized because its property '[iterator]' cannot be serialized.
-                 ~~~~~
-!!! error TS4118: The type of this node cannot be serialized because its property '[unscopables]' cannot be serialized.

```

Comment:
It's probably highly related to what I've mentioned in the other comment [here](https://github.com/microsoft/TypeScript/pull/51650/files#r1032846591). Those symbols~ should be serializable though (IMHO), so it looks like a good change